src/caret_analyze/architecture/struct/callback.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from abc import ABCMeta, abstractmethod
+from typing import Optional, Tuple
+
+from ...common import Summarizable, Summary
+
+from ...value_objects import (CallbackStructValue, CallbackType,
+                              SubscriptionCallbackStructValue,
+                              TimerCallbackStructValue)
+
+
+class CallbackStruct(Summarizable, metaclass=ABCMeta):
+    """Callback value base class."""
+
+    def __init__(
+        self,
+        node_name: str,
+        symbol: str,
+        subscribe_topic_name: Optional[str],
+        publish_topic_names: Optional[Tuple[str, ...]],
+        callback_name: str,
+    ) -> None:
+        self._node_name = node_name
+        self._callback_name = callback_name
+        self._symbol = symbol
+        self._subscribe_topic_name = subscribe_topic_name
+        self._publish_topic_names = publish_topic_names
+
+    @property
+    def node_name(self) -> str:
+        """
+        Get node name.
+
+        Returns
+        -------
+        str
+            node name
+
+        """
+        return self._node_name
+
+    @property
+    def symbol(self) -> str:
+        """
+        Get callback symbol name.
+
+        Returns
+        -------
+        str
+            callback symbol name
+
+        """
+        return self._symbol
+
+    @property
+    def callback_name(self) -> str:
+        """
+        Get callback name.
+
+        Returns the backslash is redundant between brackets
```

Comment:
docstringがおかしくなっている気がします。

```suggestion
        Returns
```

Reply:
[d20e7f5](https://github.com/tier4/CARET_analyze/pull/84/commits/d20e7f52d6e50606b632ea1bc749c272504accc6)にて修正を行いました。